src/main.jsx

```diff
@@ -15,10 +15,6 @@ const router = createBrowserRouter([
     children: [
       {
         path: "/",
```

Comment:
If the path for the book list was modified, the path for the book detail should be as well